cli/internal/otel/receiver.go

```diff
@@ -135,7 +145,7 @@ func newMetricConsumer(metricsFile *os.File, quit chan any, wg *sync.WaitGroup)
 		}
 	}()
 
-	return func(ctx context.Context, metric pluginMetric) {
+	consumerFunc := func(ctx context.Context, metric pluginMetric) {
```

Comment:
Not just `return func(..) { .. }, func(..) { .. }` to preserve indentation for untouched code.